components/playlist/browser/resources/components/playlistFolder.tsx

```diff
@@ -114,6 +116,57 @@ function EditActionsContainer ({
   )
 }
 
+const StyledEmptyFolderContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: 1 0 0;
+  align-self: stretch;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+`
+
+const StyledEmptyFolderMessageContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: ${spacing['2Xl']};
+  flex: 1 0 0;
+`
+
+const StyledEmptyFolderIcon = styled.div`
+  content: url(${`${window.location.origin}/${EmptyFolderIcon}`});
```

Comment:
Nit: can we do `/${EmptyFolderIcon}` here or is there something that means the relative URL doesn't work properly :laughing: 
```suggestion
  content: url(/${EmptyFolderIcon});
```